ci/env/install-minimal.sh

```
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -xe
-
```

this is not fixing the issue, just suppressing the error blindly.